Assets/Python/CityNameManager.py

```diff
@@ -276,6 +277,7 @@ def revertCommunistNames(iCiv):
 	'Ayutthaya'		:	'Bangkok',
 	'York'			:	'Toronto',
 	'Barara'		:	'Addis Ababa',
+	'Dongrae'		:	'Busan',

```

Comment:
What is the significance of Dongrae?

Reply:
Dongrae was the name of the larger district, and Busan was part of it.
After Japan colonized Korea they named the entire area as the city of Busan,
and Dongrae became a part of the city.
This is still the case after the independence of Korea,
so with this update Dongrae will change name to Busan in-game if
a) another nation acquires it from Korea
b) Korea enters the Industrial Era